apollos/core/blocks/nav/Link.jsx

```diff
@@ -74,7 +74,10 @@ export default class NavLink extends Component {
   }
 
   render () {
-    const iconClasses = `${this.props.navItem.icon} display-block`;
+    let icon = this.props.navItem.icon
+    if (this.props.navItem.isActive && this.props.navItem.isActive(this.props) && this.props.navItem.activeIcon) { icon = this.props.navItem.activeIcon }
+
```

Comment:
This is a little long to me. Can we add some line breaks and some semicolons? Something like this?

```javascript
let icon = this.props.navItem.icon;
if (
    this.props.navItem.isActive && 
    this.props.navItem.isActive(this.props) && 
    this.props.navItem.activeIcon
) { 
  icon = this.props.navItem.activeIcon;
}
```